app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -350,12 +366,16 @@ public void onStopTrackingTouch(SeekBar seekBar) {
         });
 
         binding.mainImage.setOnClickListener(view -> onMainImageTap());
+
```

Comment:
Maybe I'm weird, but I quite like the `setOnClickListener` and `setOnLongClickListener` of the same object being directly below each other :)

```suggestion
```